src/pages/ReimbursementAccount/BeneficialOwnersStep.tsx

```diff
@@ -37,7 +37,7 @@ function BeneficialOwnersStep({onBackButtonPress}: BeneficialOwnersStepProps) {
     const [reimbursementAccountDraft] = useOnyx(ONYXKEYS.FORMS.REIMBURSEMENT_ACCOUNT_FORM_DRAFT);
 
     const companyName = reimbursementAccount?.achData?.companyName ?? '';
-    const policyID = reimbursementAccount?.achData?.policyID ?? '-1';
+    const policyID = String(reimbursementAccount?.achData?.policyID);
```

Comment:
I'm not sure about this change here, what if `reimbursementAccount?.achData?.policyID` is undefined or null

Reply:
Im not fan of this changed either. But this is example straight out of Style.md guidelines - https://github.com/Expensify/App/blob/main/contributingGuides/STYLE.md#default-value-for-inexistent-ids

// BAD
const accountID = report?.ownerAccountID ?? -1;
const policyID = report?.policyID ?? '-1';
const managerID = report ? report.managerID : 0;

// GOOD
const accountID = report?.ownerAccountID ?? CONST.DEFAULT_NUMBER_ID;
const policyID = report?.policyID;
const managerID = report ? report.managerID : CONST.DEFAULT_NUMBER_ID;

Reply:
But I don't see the guideline about casting these id to string, can we just use `const policyID = reimbursementAccount?.achData?.policyID` instead?

Reply:
Sure

Reply:
Thank you. All good. Let me know when the BE is ready 😄